.gitmodules

```diff
@@ -5,3 +5,6 @@
 [submodule "lib/ds-test"]
 	path = lib/ds-test
 	url = https://github.com/dapphub/ds-test
+[submodule "lib/prb-math"]
```

Comment:
since you're using openzeppelin, it should be here too. As I've mentioned, mixing packages.json and libs is just confusing because it's not clear where packages are coming from. The other approach is to move these to packages.json and just use that approach.

Reply:
I don't know how to tell. I don't see a remappings file, so I wouldn't know how anything would map @prb/math to lib/prb-math/src, so I'm guessing you're using the one in node modules.

Reply:
This is still unresolved. Wanted to focus on the main contract and tests first, but will figure this out before the audit.

Reply:
I asked the JB devs and they said their template that I forked from was meant to use both and there was a reason, but they have not gotten back to me with what the specific reason was yet. But you can see in their template readme that I forked from that they explicitly mention using both (https://github.com/jbx-protocol/juice-contract-template). So, unless you have a strong opinion on this, I would rather keep it as is, to avoid possibly breaking something, if that’s cool.

Reply:
No worries. Do what you need to do. I think I found the logic in the docs:

> If the dependency you would like to install does not have an up-to-date NPM package, use `forge install` ... Forge manages dependencies using git submodules.

That logic is bad to me, but I'll just agree to disagree with the JB folks. My reasoning is that if there's not an updated NPM package, but Forge is getting the package from _somewhere_, where and why do we think that Forge's package is better than the non-updated one? Typically bad move to use multiple package managers because the libraries will start to conflict.